src/viam_mlmodelservice_triton_impl.cpp

```diff
@@ -98,7 +98,7 @@ class Service : public vsdk::MLModelService, public vsdk::Stoppable, public vsdk
         // drain.
     }
 
-    void stop(const vsdk::AttributeMap& extra) noexcept final {
+    void stop(const vsdk::ProtoStruct& extra) noexcept final {
```

Comment:
Changes in this file are all associated with https://github.com/viamrobotics/viam-cpp-sdk/pull/301. I made these changes mechanically and need to verify I've done it right.